cmd/appliance/upgrade/prepare.go

```diff
@@ -731,8 +819,9 @@ func prepareRun(cmd *cobra.Command, args []string, opts *prepareUpgradeOptions)
 
 const prepareUpgradeMessage = `PREPARE SUMMARY
 
-1. Upload upgrade image {{.Filepath}} to Controller
-2. Prepare upgrade on the following appliances:
+1. Bundle and upload LogServer docker image if needed
```

Comment:
Should this always be shown, even though there are on log server in the collective?

Reply:
No this should only be shown if there is a log server being upgraded.
We should make sure the whole flow is only triggered in that case.

Reply:
Probably not, but takes a bit of work to avoid it, so I guess the question would be if it's a deal breaker before merging?

Reply:
I think so yes, this will confuse people.
It also needs to be only shown for upgrades to 6.2.0+ (with a logserver being upgraded).

Reply:
Ok, I'll give it some time then before merge